### PR TITLE
220 Update Wallet Detail page & Collective page for active donation

### PR DESCRIPTION
**Pull request by sirpy**
Screenshot 2024-11-26 at 13 13 22
Screenshot 2024-11-26 at 13 12 59
Screenshot 2024-11-26 at 13 12 48
Screenshot 2024-11-26 at 13 11 34
Screenshot 2024-11-26 at 13 11 30
Screenshot 2024-11-26 at 13 10 37
Screenshot 2024-11-26 at 13 10 31

### CODE REVIEW

File: packages/app/src/components/WalletCards/styles.ts
Comment (L03TJ3) [inline packages/app/src/components/WalletCards/styles.ts:1]: not using native-base theming

File: packages/app/src/components/ActiveStreamCard.tsx
Comment (L03TJ3) [inline packages/app/src/components/ActiveStreamCard.tsx:1]: If native-base components are used, native-base theming should be applied to keep consistent

**Review by L03TJ3 — CHANGES_REQUESTED** [review]

**Comment by L03TJ3** [comment]
native-base theming and components are not used

File: packages/app/src/components/WalletCards/WalletDonatedCard.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <Text style={styles.info}>{userName} has donated</Text>
+      <View style={styles.row}>
+        <Text style={styles.bold}>G$ </Text>
+        <GoodDollarAmount
+          style={styles.totalReceived}
+          lastDigitsProps={{ style: { fontSize: 18, fontWeight: '300', lineHeight: 33 } }}
+          amount={donationsFormatted || '0'}
+        />
+      </View>
+      <Text style={styles.formattedUsd}>= {donationsUsdValue} USD</Text>
```
[the remark this answers is not in the document]
Comment (sirpy) [reply packages/app/src/components/WalletCards/WalletDonatedCard.tsx:32]: added default 0

**Review by L03TJ3 — APPROVED** [review]

**Pull request merged**